docs(fuse-vhd): document cli file restoration (#8355)
vatesfr/xen-orchestra: @vates/fuse-vhd/README.md

`xo-server-db ls remote`

The `vhdPathInRemote` follows a file structure described [here](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/backups/docs/VM%20backups/README.md).

## Restore a file from a VHD using `fuse-vhd` CLI

1. Mount a VHD to the Filesystem (see **Usage**).
2. Verify the VHD is Correctly Mounted:

`mount | grep fuse`

3. Get the `START` and `SIZE` of the disk whose file you want to restore. Multiply `START` by 512 (block size) to get the offset value. (`mountedVhdPath` is the `mountPoint` value from earlier, followed by `/vhd0`, for example). Depending on the partition type, you may need to do some additional setup to discover the partitions.

`partx --bytes --output=NR,START,SIZE,NAME,UUID,TYPE --pairs <mountedVhdPath>`

4.  Mount the disk. Depending on the partition type, you may need to add some additionnal options. The `norecovery` option is used for ext3/ext4/xfs file systems, otherwise remove this option.

`mount --options=loop,ro,norecovery,sizelimit=<SIZE>,offset=<START*512>  --source=<mountedVhdPath> --target=<diskMountPoint>`

> 5. Copy Your Files from the mounted partition and then unmount the partition.

Reviewer: ```suggestion
5. Copy your files from the mounted partition, then unmount the partition.
```

xo-fuse-vhd <remoteUrl> <vhdPathInRemote> <mountPoint>
```

> remoteUrl can be found by using cli in `@xen-orchestra/fs` , for example a local remote will have a url like `file:///path/to/remote/root`, or with command:

Reviewer: ```suggestion
The `remoteUrl` can be found using the CLI in `@xen-orchestra/fs`. For example, a local remote will have a URL like `file:///path/to/remote/root`. Alternatively, you can use the following command:
```

> remoteUrl can be found by using cli in `@xen-orchestra/fs` , for example a local remote will have a url like `file:///path/to/remote/root`, or with command:

Reviewer 2: ```suggestion
The `remoteUrl` can be found using the command line from the xoa, with the command xo-server-db ls remote, and then picking the right `url` property
```

```
xo-server-db ls remote
```

> ## Restore a file from a VHD using `fuse-vhd` CLI

Reviewer: ```suggestion
## Restore a file from a VHD using `fuse-vhd` CLI

1. Mount a VHD to the Filesystem (see **Usage**).
2. Verify the VHD is Correctly Mounted:

`mount | grep fuse`

3. Get the `START` and `SIZE` of the disk whose file you want to restore. Multiply `START` by 512 (block size) to get the offset value. (`mountedVhdPath` is the `mountPoint` value from earlier, followed by `/vhd0`, for example).

`partx --bytes --output=NR,START,SIZE,NAME,UUID,TYPE --pairs <mountedVhdPath>`

4.  Mount the disk:

`mount --options=loop,ro,norecovery,sizelimit=<SIZE>,offset=<START*512>  --source=<mountedVhdPath> --target=<diskMountPoint>`

5. Copy Your Files from the mounted partition and then unmount the partition.

```

Reviewer 2: this will only work on the partition type that can be mounted without options. For example, LVM setups need additionnal setup to discover and mount them.
